azure/iac/master-compliance-test.json

```diff
@@ -2524,7 +2524,7 @@
                             "id": "PR-AZR-0125-ARM",
                             "eval": "data.rule.protocol",
                             "message": "data.rule.protocol_err",
-                            "remediationDescription": "",
+                            "remediationDescription": "Make sure ApplicationGateway has (properties.httpListeners.properties.protocol='Https')",
```

Comment:
**See my validation message at line 2432 (for id PR-AZR-0107-ARM)**. make the description format like this one.

Reply:
fixed